src/main/java/audio/Util.java

```diff
@@ -58,4 +58,8 @@ public static byte[] convertShortToByte(short[] data) {
         }
         return outBuffer;
     }
+
+    public static double findAverage(double[] preOut) {
```

Comment:
This does the same as Maths.mean() that was just added in #160 . One of them should be removed, right?